components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.scss

```diff
@@ -210,11 +210,42 @@ chef-alert {
   }
 
   .healthcheck-content {
+    position: relative;
     border-radius: 0 0 2px 2px;
     background-color: var(--chef-primary-dark);
     color: var(--chef-white);
-    padding: 8px;
+    padding: 8px 8px 40px 8px;
     overflow-wrap: break-word;
+    max-height: 94px;
+    overflow: hidden;
+    transition: .25s max-height;
+
+    &.active {
+      max-height: 540px; // temp placeholder
```

Comment:
we only expand as far as the text needs to go by using max-height instead of height, tapping out at 540px max because we will only ever show 1024 characters (need to confirm with applications)